src/Microsoft.PowerShell.SDK/Microsoft.PowerShell.SDK.csproj

```diff
@@ -16,11 +16,11 @@
 
   <ItemGroup>
     <!-- the following package(s) are from https://github.com/dotnet/fxdac -->
-    <PackageReference Include="System.Data.SqlClient" Version="4.7.0-preview8.19405.3" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.7.0-preview9.19421.4" />
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="System.IO.Packaging" Version="4.6.0-preview8.19405.3" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.6.0-preview8.19405.3" />
-    <PackageReference Include="System.Text.Encodings.Web" Version="4.6.0-preview8.19405.3" />
+    <PackageReference Include="System.IO.Packaging" Version="4.6.0-preview9.19421.4" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.6.0-preview9.19421.4" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.6.0-preview9.19421.4" />
     <!-- the following package(s) are from https://github.com/dotnet/wcf -->
     <PackageReference Include="System.ServiceModel.Duplex" Version="4.6.0-preview8.final" />
```

Comment:
Why aren't we updating this and some packages below?

Reply:
Thanks, I've updated those `*.final` packages now as well